### DISCUSSION TITLE
[Feat]: Build questions

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5678] [Q&A]
<img width="933" alt="image" src="https://user-images.githubusercontent.com/56351544/196949992-e3400ec0-23c5-46d1-9762-50255d537312.png">
I was stuck here for a long time when executing 'yarn'. What is the reason? Is there any solution

**Comment by chenhuanguang** [comment]
![image](https://user-images.githubusercontent.com/56351544/196950527-e2ed8271-022a-470d-855a-ead7a7e1cc12.png)

**Reply by jsjoeio** [reply]
It has to install all the dependencies for each extension in vscode so it will take a long time for the first run
